open-craft/opencraft
# [BB-1484] Enable moving media assets from SWIFT to S3

This PR adds code changes which allow using S3 as the media storage backend.

**Testing instructions**:
- Deploy this PR to Ocim stage.
- Deploy the updated `.env` file using the changes in the related `ansible-secrets` PR.
- Restart Ocim.
- Test and verify that the media files are uploaded to the configured S3 bucket and the appservers are able to access the media files during their provisioning and that the correct logo and favicon images are set in the instance.
